Fix service check clarification (#8395)
DataDog/integrations-core: mysql/assets/SERVICE_CHECK_CLARIFICATION.md

## On a slave host:

* If the SQL and the IO threads are running, returns OK

>     * In the case of version > 5.7.0 Slave_running need to also be on

Reviewer: ```suggestion
    * In the case of version > 5.7.0, `Slave_running` is required to be ON.
```

small nit to match the code comment
